README.md

```diff
@@ -337,9 +365,17 @@ This may take a few minutes to finish.
 
 ### Linked Open Data (LOD) Authorities
 
+You will need to add gems that process the type of linked data returned for the authorities you use.  A gem that covers 
+all formats is [ruby-rdf/linkeddata](https://github.com/ruby-rdf/linkeddata).  This gem is included in QA for development 
+and testing of QA, but is not automatically included in the released gem.  Additionally, it is unlikely that you will need 
+all the formats included by that gem.  You may want to select only those gems that are for the formats you need supported.  
+See all gems in [linkeddata.gemspec](https://github.com/ruby-rdf/linkeddata/blob/develop/linkeddata.gemspec).
```

Comment:
While I know to add "gem 'linkeddata'" to my gemfile, spelling out the step explicitly may help someone newer to Rails who just wants to get it up and running. Furthermore, the more advanced configuration explanation might be better in its own wiki page since even I don't know exactly what set of gems I need from linkeddata for this to work and more guidelines of "include X,Y,Z as a base and then A for turtle, B for json-ld, etc" would be useful.